sdks/go/test/integration/primitives/pardo.go

```diff
@@ -156,3 +159,26 @@ func splitByName(key string, vals []string, a, j, d func(string)) {
 		emitter(val)
 	}
 }
+
+// ParDoMultiOutput tests that a DoFn can access PipelineOptions.
+func ParDoPipelineOptions() *beam.Pipeline {
+	// Setup some fake flags
+	flag.String("A", "", "Flag for testing.")
+	flag.String("B", "", "Flag for testing.")
+	flag.String("C", "", "Flag for testing.")
+	flag.CommandLine.Parse([]string{"--A=123", "--B=456", "--C=789"})
+
+	p, s := beam.NewPipelineWithRoot()
+
+	in := beam.Create(s, 1)
```

Comment:
To kick off a pipeline without using a create, use beam.Impulse() to get a PCollection<[]byte>. Then you can ignore the value in the downstream dofn by calling it `_`.

Reply:
Neat! Done